src/policy/rbf.cpp

```diff
@@ -33,13 +39,14 @@ RBFTransactionState IsRBFOptIn(const CTransaction &tx, CTxMemPool &pool)
 
     // If all the inputs have nSequence >= maxint-1, it still might be
     // signaled for RBF if any unconfirmed parents have signaled.
-    uint64_t noLimit = std::numeric_limits<uint64_t>::max();
+    const uint64_t noLimit = std::numeric_limits<uint64_t>::max();
     std::string dummy;
-    CTxMemPoolEntry entry = *pool.mapTx.find(tx.GetHash());
+    const CTxMemPoolEntry entry = *pool.mapTx.find(tx.GetHash());
     pool.CalculateMemPoolAncestors(entry, setAncestors, noLimit, noLimit, noLimit, noLimit, dummy, false);
 
-    for (CTxMemPool::txiter it : setAncestors) {
-        if (SignalsOptInRBF(it->GetTx())) {
+    for (const CTxMemPool::txiter it : setAncestors) {
```

Comment:
Should be `const` reference instead?

Reply:
txiter is already const_txiter, no need to change anything here